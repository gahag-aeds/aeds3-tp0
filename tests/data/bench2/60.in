60 30
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 
Sub 38 57
Sum 9 28
Sub 30 49
Sum 10 29
Sub 24 43
Min 5 24
Sub 36 55
Max 8 27
Sub 28 47
Sum 34 53
Add 11 30
Min 18 37
Add 15 34
Min 7 26
Add 23 42
Min 24 43
Sub 28 47
Max 18 37
Sub 12 31
Min 14 33
Add 21 40
Max 7 26
Add 11 30
Sum 29 48
Sub 24 43
Sum 13 32
Sub 28 47
Sum 39 58
Add 39 58
Min 40 59
