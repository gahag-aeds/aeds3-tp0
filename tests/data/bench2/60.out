1008
1031
10
85
836
2
12
3
94
9
88
827
995
680
-4
